themes/bootstrap3/js/check_item_statuses.js

```diff
@@ -1,232 +1,260 @@
-/*global Hunt, VuFind */
+/*global Hunt, StatusAjaxQueue, VuFind */
 
-VuFind.register('itemStatuses', function ItemStatuses() {
+VuFind.register("itemStatuses", function ItemStatuses() {
   function formatCallnumbers(callnumber, callnumber_handler) {
-    var cns = callnumber.split(',\t');
+    var cns = callnumber.split(",\t");
```

Comment:
I think it's possible to set up eslint with quote preferences (at least, I'm pretty sure I remember seeing it switch quotes around in other projects), but maybe we haven't done that here. I don't have strong feelings -- this is just a huge amount of code to review, and I'm looking for anything that might reduce the mental load necessary to process it all. ;-)

In any case, since this is not yet urgent, I'll keep focusing on the 8.1 release items for now, and will circle back around to this one when there's a bit more free time to roll up my sleeves and dig in.

Reply:
Agreed but there are no extraneous files in the PR after rolling back the eslint changes to js/lib/.

Reply:
Yes, that's a good start! And speaking of js/lib/, did you see the thread above about lib vs. vendor?